cleanlab/experimental/datalab/examples/datalab_usage.ipynb

```diff
@@ -0,0 +1,383 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab = Datalab(data, label_name=\"y\")\n",
+    "knn = NearestNeighbors(n_neighbors=3, metric=\"euclidean\")\n",
+    "issue_types = {\n",
+    "    \"outlier\": {\"ood_kwargs\": {\"params\": {\"knn\": knn}}},\n",
+    "    \"near_duplicate\": {\"metric\": \"euclidean\"},\n",
+    "}\n",
+    "\n",
+    "lab.find_issues(\n",
+    "    pred_probs=pred_probs,\n",
+    "    features=\"X\",\n",
+    "    issue_types=issue_types,\n",
+    ")\n",
+    "lab.report(k=10, verbosity=2)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice how last 5 examples correspond to the synthetic outliers we added to the data, which includes the two points that had a lower cosine distance in the previous outlier-issue checks."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding a custom IssueManager"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.experimental.datalab.issue_manager import IssueManager\n",
+    "from cleanlab.experimental.datalab.factory import register as register_issue_manager\n",
+    "\n",
+    "\n",
+    "def scoring_function(idx: int) -> float:\n",
+    "    if idx == 0:\n",
+    "        # Zero excluded from the divisibility check, gets the highest score\n",
+    "        return 1\n",
+    "    rem = idx % 13\n",
+    "    inv_scale = idx // 13\n",
+    "    if rem == 0:\n",
+    "        return 0.5 * (1 - np.exp(-0.1*(inv_scale-1)))\n",
+    "    else:\n",
+    "        return 1 - 0.49 * (1 - np.exp(-inv_scale**0.5))*rem/13\n",
+    "\n",
+    "\n",
+    "@register_issue_manager\n",
+    "class SuperstitionIssueManager(IssueManager):\n",
+    "    \"\"\"A custom issue manager that keeps track of issue indices that\n",
+    "    are divisible by 13.\n",
+    "    \"\"\"\n",
+    "    issue_name: str = \"superstition\"\n",
+    "\n",
+    "    def find_issues(self, div=13, **_) -> None:\n",
+    "        ids = self.datalab.issues.index.to_series()\n",
+    "        issues_mask = ids.apply(lambda idx: idx % div == 0 and idx != 0)\n",
+    "        scores = ids.apply(scoring_function)\n",
+    "        self.issues = pd.DataFrame(\n",
+    "            {\n",
+    "                f\"is_{self.issue_name}_issue\": issues_mask,\n",
+    "                self.issue_score_key: scores,\n",
+    "            },\n",
+    "        )\n",
+    "        summary_score = 1 - sum(issues_mask) / len(issues_mask)\n",
+    "        self.summary = self.get_summary(score = summary_score)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lab.find_issues(issue_types={\"superstition\": {}})\n",
```

Comment:
I'm unsure if I'm doing this wrong or if this is a bug - if I wanted to specify a different "superstition number" (or in the more general setting, pass any kwargs to the `.find_issues` method), this should be the code I call right? 

```
lab.find_issues(issue_types={"superstition": {"div": 4}})
```

Currently calling this is giving me an error:
```
TypeError: __init__() got an unexpected keyword argument 'div'
```

If this is not the intended way to call it I'd recommend stating somewhere what the intended way is (it's non-obvious to me and hence would probably be the same for others)

Reply:
I've added var-kwargs support to the `IssueManager.__init__()` method, so I think it should work with the latest changes on the branch.

Care to try again @huiwengoh ?

Reply:
Calling the above code no longer results in a error, but it seems like the issue itself ignores the kwargs passed in. For this example, even if i pass in `{"div": 4}` , the superstition issue manager still does the check with 13 (the default arg).

Reply:
I cannot reproduce that error.